src/main/java/frc/robot/subsystems/grabber/GrabberInputs.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.grabber;
+
+import edu.wpi.first.epilogue.Logged;
+
+@Logged
+public class GrabberInputs {
+    public double currentVelocity;
+    public double currentAppliedVoltage;
+    public double currentPosition;
+
+    public double motorTemperature;
+    public double currentDraw;
+
+    public boolean beamBreakState;
```

Comment:
We can use a name that is less hardware specific. Calling it something like `gamePieceDetected` would work.